pegasus/forms/hoc_survey_prize_2015.rb

```diff
@@ -9,9 +9,10 @@ def self.normalize(data)
     result
   end
 
-  def self.process(data)
```

Comment:
Also I called the parameter row because I was copying another [form](https://github.com/code-dot-org/code-dot-org/blob/1b19ed45e17897881d00fcdd931c489aea9d05e4/pegasus/forms/professional_development_workshop.rb#L59-L59), but I think 'form' is more clear so I'll update that.